apps/web/src/app/api/happening/[slug]/register/route.ts

```diff
@@ -152,6 +178,25 @@ export const POST = withSession(
             },
           });
 
+          const answersToSave = [];
+
+          for (var happeningQuestion of happening.questions) {
```

Comment:
Her kan men heller bruker `const` istedenfor `var`. Generelt alltid bruker `const`. Om `const` ikke funker, så bruker man oftest `let`.

https://dev.to/paritho/3-reasons-to-use-var-in-javascript-1hoe